common/djangoapps/student/models.py

```diff
@@ -1196,6 +1196,13 @@ def users_enrolled_in(cls, course_id):
         )
 
     @classmethod
+    def enrolled_and_dropped_out_users(cls, course_id):
```

Comment:
It's returning the users both enrolled and unenrolled, i.e. without a filter on the status.

The normal usage of the english word "or" is actually XOR in the logical sense, meaning either A or B but not both (e.g. would you like tea or coffee : either tea or coffee). "enrolled_or_dropped_out_users" would mean either the enrolled or the unenrolled but not both, which is not intended here.

Should we change it to "or" anyway?